src/traversing.js

```diff
@@ -168,10 +159,18 @@ jQuery.each({
 		return elem.contentDocument || jQuery.merge( [], elem.childNodes );
 	}
 }, function( name, fn ) {
+	var // Sorting (which implicitly de-dupes) is required for non-adjacent traversals
+		sort = name !== "prev" && name !== "next",
```

Comment:
Can we just do these inside the `if` below to avoid doing them if `this.length === 1`? We wouldn't even need the variables that way.